chal8.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    Letters: <input id = "characters"><br>
+    Word: <input id = 'word'><br>
+    <button onclick="start()"></button>
+    <p id = "storage"></p>
+    <script type="text/javascript">
+      function start(){
```

Comment:
Looks like it works, but I would try to redo it using ES6 syntax. There are lots of `var`s in the code that can be updated.